includes/wikia/tests/WikiaViewTest.php

```diff
@@ -39,7 +39,6 @@ public function testCreatingNewFromControllerAndMethod( $controllerName, $method
 	public function setTemplateDataProvider() {
 		return array(
 		 array( true, 'Test' ),
-		 array( true, 'Oasis' ), // tmp, while modules still exists
```

Comment:
The unit tests were failing because of this line.  The test in question was purposefully faking out controller entries in ```$wgAutoloadClasses``` for some unknown reason here:

  https://github.com/Wikia/app/blob/SOC-450/includes/wikia/tests/WikiaViewTest.php#L49-L62

Owen and I tried to figure out what this god awful test was doing and failed.  Line 41 above works because they defined a fake controller in their fake test path, but line 42 failed because there was no ```OasisController.class.php``` at the fake path.  Given that it doesn't test anything useful, we removed it.